packages/adapters/bitcoin/src/adapter.ts

```diff
@@ -79,6 +124,25 @@ export class BitcoinAdapter extends AdapterBlueprint<BitcoinConnector> {
       )
     )
 
+    if (accounts && accounts.length > 1) {
```

Comment:
Why do we expect accounts to be more than 1? Wouldn't single account be enough? I mean some wallets might return signle account maybe?

Reply:
Good point, at least 1 account should suffice